File: examples/plainJava/main/NetworkRequest.java
import netware.client.RequestClient;

public class NetworkRequest {

    public static void main(String[] args) {

        RequestClient requestClient = new RequestClient(
                "http://localhost:3000",
                "GET"
        ).build();

        if (requestClient.isSuccess()) {
            System.out.println(requestClient.response().getLog(
                    false
            ));
        } else {
            System.out.println(requestClient.error().getLog());
        }
    }
}

File: examples/plainKotlin/main/NetworkRequest.kt
import netware.client.RequestClient
import netware.client.requestExtenstions.RequestClientHeaders

fun networkRequest() {

    val requestClientHeaders = RequestClientHeaders()
    requestClientHeaders.add("x-api-key", "9CF4TaIX629gKf5Er767bR149OmVy71V5Pr5CI0v")

    val requestClient = RequestClient(
        url = "http://localhost:3000",
        method = "GET",
        headers = requestClientHeaders.getHeaders()
    ).build()

    if (requestClient.isSuccess) {
        println(
            requestClient.response().getLog(
                formatResponse = true
            )
        )
    } else {
        println(requestClient.error().getLog())
    }
}

fun main() {

    networkRequest()
}